testsuite/tests/ui/tokens/test_combinations.py

```diff
@@ -0,0 +1,98 @@
+"""
+Tests for combination of token scopes
+"""
+
+import pytest
+
+from testsuite.ui.views.admin.settings.tokens import Scopes, TokenNewView
+from testsuite.utils import blame
+
+
+@pytest.fixture(scope="module")
+def token(custom_admin_login, navigator, request, threescale, permission):
+    """
+    Create token with all scopes
+    """
+    custom_admin_login()
+    new = navigator.navigate(TokenNewView)
+    name = blame(request, "token")
+    token = new.create(name,
+                       [Scopes.BILLING.value, Scopes.MANAGEMENT.value, Scopes.POLICY.value, Scopes.ANALYTICS.value],
+                       permission[0])
+
+    def _delete():
+        token = list(filter(lambda x: x["name"] == name, threescale.access_tokens.list()))[0]
+        threescale.access_tokens.delete(token.entity_id)
+
+    request.addfinalizer(_delete)
+    return token
+
+
+def test_read_service(token, api_client):
+    """
+    Request to get list of services should have status code 200
+    """
+
+    response = api_client("GET", '/admin/api/services', token)
+    assert response.status_code == 200
+
+
+def test_create_account_user(account, token, api_client, request, permission):
```

Comment:
where are these created objects deleted?

Reply:
account's user is deleted together with account during account fixture cleanup